packages/scalecube-microservice/src/api/ProxyOptions.ts

```diff
@@ -1,6 +1,16 @@
 import { ServiceDefinition, Router } from '.';
 
+/**
+ * The options that are used for the creation of the proxy for the specific microservice container
```

Comment:
```
/**
 * ProxyOptions - The options that are used for the creation of the proxy for the specific microservice container
 *
 * @param router - Router logic
 *
 * @param serviceDefinition - Service definition for the required service
 *
 */
```

Reply:
Custom router specifies the logic of choosing the appropriate remoteService